packages/fx-core/src/common/constants.ts

```diff
@@ -34,3 +34,9 @@ export class FeatureFlagName {
   static readonly ArmSupport = "TEAMSFX_ARM_SUPPORT";
   static readonly BicepEnvCheckerEnable = "TEAMSFX_BICEP_ENV_CHECKER_ENABLE";
 }
+
+export class ArmDeploymentStatus {
```

Comment:
I don't suggest we define another status class for @azure/arm-resource ProvisioningState. We will have to maintain it once the lib upgrades. Why don't we use the type defined in the lib directly?

Reply:
It is fixed. Thanks.